lib/services/service_brokers/v2/client.rb

```diff
@@ -79,15 +79,35 @@ def fetch_service_instance_state(instance)
       result.merge(parsed_response.symbolize_keys)
     end
 
+    def create_service_key(key, arbitrary_parameters: {})
```

Comment:
@dsabeti, yes, it's duplicate of these two methods, one thing I'm thinking is create-service-key is just reusing the name of "binding", is it possible that in the future, broker will support two kinds of endpoint which will support create-service-key and service-binding separately? this may make the code more clear. if you think it's better to reuse the "bind" method, I'll change it and resubmit this one, thanks